android/src/main/AndroidManifest.xml

```diff
@@ -6,7 +6,8 @@
         </intent>
     </queries>
 
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"
```

Comment:
I think this change is causing the `User did not grant library permission` that some people is having on android devices. @ivpusic @sean5940